Sources/SwiftParser/Syntax+StringInterpolation.swift

```diff
@@ -1,6 +1,43 @@
 @_spi(RawSyntax)
 import SwiftSyntax
 
+fileprivate class Indenter: SyntaxRewriter {
```

Comment:
I'm not that familiar with `SwiftParser` yet, so apologies if I'm missing something. Why do we need an indentation rewriter within the parser? Don't Swift's multi-line string literals already handle indentation properly?

Reply:
The problem is that if you have

```swift
let nested = """
first
second
"""

let wrapping = """
  \(nested)
"""

print(wrapping)
```

you get 

```
  first
second
```

where `second` is not indented. Basically all I’m doing here is indenting all lines in the string interpolation to the current indentation level.